packages/tools/devtools/devtools-example/src/Container.ts

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ModelContainerRuntimeFactory } from "@fluid-example/example-utils";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { IContainer } from "@fluidframework/container-definitions";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+import { CollaborativeDevtoolExampleAppObjects } from "./FluidObject";
+
+/**
+ * The CollaborativeDevtoolExampleAppModel is the model for our Fluid object.
```

Comment:
Nit: I generally recommend not referring to the thing being documented by name in the comments. The comment is attached to the object, so calling out the name here isn't really necessary.

Reply:
(same with some of the other type comments in this file)

Reply:
Fixed